openwisp_controller/config/base/config.py

```diff
@@ -318,6 +318,17 @@ def enforce_required_templates(cls, action, instance, pk_set, **kwargs):
                     *required_templates.order_by('name').values_list('pk', flat=True)
                 )
 
+    @classmethod
+    def certificate_renewed(cls, instance, created, **kwargs):
```

Comment:
I'd change this to `certificate_updated` to make it clear that we're not really checking if it's renewed but we just look for any change.